imports/components/@enhancers/security-roles/__tests__/index.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable */
+
+import { Component } from "react";
+import { shallow } from "enzyme";
+import { shallowToJson } from "enzyme-to-json";
+import { SECURITY_ROLES_QUERY, canSee, authorized } from "../";
+
+it("has a valid query", () => {
+  expect(SECURITY_ROLES_QUERY).toMatchSnapshot();
```

Comment:
@delianides let's just pretty print this query instead of snapshotting the generated AST. Like https://github.com/NewSpring/Holtzman/pull/1772/files#diff-86223d28ba8005f2d829c68101632715R379